cli/src/commands/extensions/api.rs

```diff
@@ -496,19 +514,22 @@ fn add_permission_args<'a>(
 }
 
 /// Return error when trying to sandbox on Windows.
-#[op]
 #[cfg(not(unix))]
-fn run_sandboxed(_process: Process) -> Result<ProcessOutput> {
+#[op2]
+#[serde]
+fn run_sandboxed(#[serde] _process: Process) -> Result<ProcessOutput> {
```

Comment:
Just have to hope I did this one right... We don't test native Windows builds anymore in CI (and I don't have a Windows host to test on)...

Maybe I can convince @furi0us333 to test this for me... Or we should discuss using [windows-xl][1] runners (at cost) periodically to test this sort of stuff.

[1]: https://docs.github.com/en/actions/using-github-hosted-runners/about-larger-runners/about-larger-runners

Reply:
I am happy to test something if needed, though I always try and push people to using WSL.

Reply:
Me too. We don't officially support Windows outside of WSL.

But we've historically made an attempt to at least build successfully on native Windows. So I was just curious if that still worked or not...